### PR TITLE
Base Control Interface framework

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

File: controlinterface/urls.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Tastypies API function
+api_resources = Api(api_name='v1/controlinterface')
+api_resources.register(api.MetricResource())
+api_resources.prepend_urls()
+
+urlpatterns = patterns('',
+    # Setting the urlpatterns to hook into the api urls
+    url(r'^api/', include(api_resources.urls)),
+    url(r'^controlinterface/$', views.index, name='index'),
+    # url(r'^controlinterface/login/$', views.login, name='login'),
```
Comment (gsvr) [inline controlinterface/urls.py]: Shall we delete this line?

File: controlinterface/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (gsvr) [inline controlinterface/tests.py:3]: Shall we add some tests?

File: controlinterface/templates/controlinterface/base_site.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{% extends "controlinterface/base.html" %}
+{% load i18n %}
+
+{% block title %}{{ title }} | {% get_site_title %}{% endblock %}
+
+{% block branding %}
+    <h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header }}</a></h1>
+{% endblock %}
+
+{% block nav-global %}{% endblock %}
```
Comment (gsvr) [inline controlinterface/templates/controlinterface/base_site.html]: eof

File: controlinterface/templates/controlinterface/base.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+         <div class="col-sm-3 col-md-2 sidebar">
+          <img src="{% static "controlinterface/img/momconnect_logo.png" %}" class="img-responsive" style="padding-left: 20px">
+          <br>
+          <br>
+          <ul class="nav nav-sidebar">
+            <li><a href="#"><span class="glyphicon glyphicon-user"></span> {% firstof user.get_short_name user.get_username|capfirst %} <span class="glyphicon glyphicon-cog pull-right"></span></a> </li>
+            <li class="active"><a href="index.html"><span class="glyphicon glyphicon-dashboard"></span> Dashboard</a></li>
```
Comment (gsvr) [inline controlinterface/templates/controlinterface/base.html]: I don't think this link works.

File: controlinterface/tests.py (hunk 2 above)
Comment (imsickofmaps) [inline controlinterface/tests.py:3]: not to this part

**Pull request merged**